config/settings.yml

```diff
@@ -480,20 +480,23 @@ flipper:
     - kmircovich@governmentcio.com
     - lauren.alexanderson@va.gov
     - lindsey.hattamer@oddball.io
+    - ltran@thoughtworks.com
```

Comment:
NIT: I'd prefer to see Flipper admins set up in a separate PR, but I don't think there's a hard and fast rule so this is fine.

Reply:
Fair enough, we'll be aware of that for future changes! Thanks.